### PR TITLE
Adds a process for adding a plugin.

### PR DESCRIPTION
**Pull request by StevenDufresne**
Here's what I'm thinking for the initial submission process.

### CODE REVIEW

**Comment by jordesign** [comment]
Looks good to me 👍

File: submit-code/readme.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Submitting Code to the Data Liberation Project
+
+You have code that can users move their data freely on the web? Awesome. This readme will provide you with the details and requirements on how to submit your code. If you get stuck along the way, [open an issue](https://github.com/WordPress/data-liberation/issues/new) or reach out in the [#data-liberation slack channel](https://wordpress.slack.com/archives/C069AKUBPHB).
```
Comment (adamwoodnz) [inline submit-code/readme.md]:
```suggestion
You have code that can help users move their data freely on the web? Awesome! This readme provides the details and requirements for how to submit your code. If you get stuck along the way, [open an issue](https://github.com/WordPress/data-liberation/issues/new) or reach out in the [#data-liberation slack channel](https://wordpress.slack.com/archives/C069AKUBPHB).
```

File: submit-code/readme.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+# Requirements
+
+You need to answer yes to the following items:
+
+- My code contains a `readme.md` in the root directory?
```
Comment (adamwoodnz) [inline submit-code/readme.md]: For consistency maybe drop the question mark, and replace with colon for the sub items
```suggestion
- My code contains a `readme.md` in the root directory:
```

**Review by adamwoodnz — APPROVED** [review]
Process sounds good to me, couple of grammatical suggestions.

Are these requirements based on prior art?

**Comment by StevenDufresne** [comment]
> Process sounds good to me, couple of grammatical suggestions.
> 
> Are these requirements based on prior art?

It depends on what you consider art. :)

I've had some informal conversations with contributors but it's not taken from anywhere specifically and likely to evolve.

**Pull request merged**